pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -43,17 +47,25 @@ class DeviceStatsMonitor(Callback):
         >>> trainer = Trainer(callbacks=[device_stats]) # doctest: +SKIP
     """
 
-    def setup(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", stage: Optional[str] = None) -> None:
+    def __init__(self, cpu_stats: bool = True) -> None:
+        self.cpu_stats = cpu_stats
+
+    def setup(
+        self,
+        trainer: "pl.Trainer",
+        pl_module: "pl.LightningModule",
+        stage: Optional[str] = None,
+    ) -> None:
         if not trainer.loggers:
             raise MisconfigurationException("Cannot use DeviceStatsMonitor callback with Trainer that has no logger.")
```

Comment:
```suggestion
            raise MisconfigurationException("Cannot use `DeviceStatsMonitor` callback with `Trainer` that has no logger.")
```
un-related to this PR, but just for consistency.